webvr_ar_extension.idl

```diff
@@ -16,6 +16,11 @@
 partial interface VRDisplay {
   sequence<VRHit> hitTest(float x, float y);
   sequence<VRPlane> getPlanes();
+  VRAnchor addAnchor(sequence<float> modelMatrix);
+  VRAnchor addAnchor(Float32Array modelMatrix);
```

Comment:
Does this mean `addAnchor` takes both an array and a Float32Array?

Reply:
Yes. It can take an array or a float32array.